internal/utils/main.go

```diff
@@ -235,10 +270,13 @@ func AddExportTypeFlags(flags *pflag.FlagSet) {
 	flags.BoolP("export-offers", "f", false, "set in order to export offer changes")
 	flags.BoolP("export-pools", "p", false, "set in order to export liquidity pool changes")
 	flags.BoolP("export-balances", "l", false, "set in order to export claimable balance changes")
+	flags.BoolP("export-contract-code", "", false, "set in order to export claimable balance changes")
+	flags.BoolP("export-contract-data", "", false, "set in order to export claimable balance changes")
+	flags.BoolP("export-config-settings", "", false, "set in order to export claimable balance changes")
 }
```

Comment:
nit: description is a copy/paste error. Do we also need short flags for the new arguments? Or should all of them be removed?